### PR TITLE
Add TestRegionalXLBCT to test ConnectionTimeout for Regional XLB Ingress

### PR DESCRIPTION
**Pull request by panslava**
- Almost a full copy of existent Timeout test
- Verified with local run, passed

### CODE REVIEW

File: cmd/e2e-test/timeout_test.go [hunk 1]
```diff
@@ -216,6 +216,100 @@ func TestILBCT(t *testing.T) {
 	}
 }
 
+func TestRegionalXLBCT(t *testing.T) {
```
Comment (bowei) [inline cmd/e2e-test/timeout_test.go:219]: TestRegionalXLBConnectionTimeout

**Comment by bowei** [comment]
lgtm apart from name of test

File: cmd/e2e-test/timeout_test.go (hunk 1 above)
Comment (spencerhance) [reply cmd/e2e-test/timeout_test.go:219]: +1

Comment (spencerhance) [reply cmd/e2e-test/timeout_test.go:219]: chatted offline, @panslava Can you add a TODO here?  Otherwise lgtm

Comment (panslava) [reply cmd/e2e-test/timeout_test.go:220]: renamed

Comment (panslava) [reply cmd/e2e-test/timeout_test.go:220]: oh, I meant, added TODO

**Pull request merged**